docs/DafnyRef/UserGuide.md

```diff
@@ -370,6 +370,49 @@ The command emits exit codes of
 
 It also takes the `--verbose` option, which then gives information about the files being formatted.
 
+The `dafny audit` command currently reports the following:
+
+* Any declaration marked with the `{:axiom}` attribute.
+This is typically used to mark that a lemma with no body (and is therefore assumed to always be true) is intended as an axiom.
+The key purpose of the `audit` command is to ensure that all assumptions are intentional and acknowledged.
+To improve assurance, however, try to provide a proof.
+
+* Any declaration marked with the `{:verify false}` attribute, which tells the verifier to skip verifying this declaration.
+Removing the attribute and providing a proof will improve assurance.
+
+* Any declaration marked with the `{:extern}` attribute that has at least one `requires` or `ensures` clause.
+If code implemented externally, and called from Dafny, has an `ensures` clause, Dafny assumes that it satisfies that clause.
+Since Dafny cannot prove properties about code written in other languages,
+adding tests to provide evidence that any `ensures` clauses do hold can improve assurance.
+The same considerations apply to `requires` clauses on Dafny code intended to be called from external code.
+
+* Any declaration marked with the `{:concurrent}` attribute.
+This is intended to indicate that the code is safe for use in a concurrent setting, but Dafny currently cannot prove that form of safety.
+The [addition of `reads` clauses to methods](https://github.com/dafny-lang/rfcs/pull/6) will be a step toward being able to prove safety in this case,
+but until it exists, careful inspection and testing are appropriate.
+
+* Any definition with an `assume` statement in its body.
+To improve assurance, attempt to convert it to an `assert` statement and prove that it holds.
+Alternatively, converting it to an `expect` statement will cause it to be checked at runtime.
+Such a definition will not be compilable unless the statement is also marked with `{:axiom}`.
```

Comment:
```suggestion
Such a definition will not be compilable unless the statement is also marked with `{:axiom}`.
Alternatively, converting it to an `expect` statement will cause it to be checked at runtime.
```

As is it sounds like you're implying an `expect` statement is not compilable without `{:axiom}`.